src/servidor/UsuarioService.ts

```
@@ -0,0 +1,21 @@
+import { BaseService } from '@/utils/BaseService.ts'
+import { Usuario } from '@/commons/interfaces.ts'
+
+class ServidorService extends BaseService {
+  constructor() {
+    super('/usuarios')
+  }
+
+  async save(servidor: Usuario): Promise<void> {
+    const response = await this.post<{ token?: string }, Usuario>(servidor)
+    if (response && response.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)